sdk/keyvault/remove-test-resources-pre.ps1

```diff
@@ -0,0 +1,79 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# IMPORTANT: Do not invoke this file directly. Please instead run eng/common/TestResources/Remove-TestResources.ps1 from the repository root.
+
+#Requires -Version 6.0
+#Requires -PSEdition Core
+
+# Use same parameter names as declared in eng/common/TestResources/Remove-TestResources.ps1 (assume validation therein).
+[CmdletBinding(DefaultParameterSetName = 'Default', SupportsShouldProcess = $true, ConfirmImpact = 'Medium')]
```

Comment:
You only have the one parameter set so don't need `DefaultParameterSetName`, nor do you actually support `ShouldProcess` so you just need this:
```suggestion
[CmdletBinding()]
```

Reply:
Thanks for looking! I went ahead and applied all of your suggestions so I will resolve these.